README.md

```diff
@@ -17,18 +17,15 @@ The [official CocoaPods website](http://cocoapods.org) has detailed instructions
 The following lines in a Podfile will install the SDK for an iOS app with a watch OS 2 extension: 
 
 ```
-source 'https://github.com/openwhisk/openwhisk-podspecs.git'
-
+install! 'cocoapods', :deterministic_uuids => false
```

Comment:
Ah, I forgot to explain this one. This is available as of Cocoapods 1.0. It is not required. It suppresses Cocoapods warnings that users would otherwise get when doing `pod install`, which may concern some users but the warnings are inconsequential. They are a result of supporting multiple OS's (iOS and watchOS) with the same pod. It is perfectly okay to just leave that line out of the Podfile. It's up to you.